docs/reference/transform/apis/get-transform-stats.asciidoc

```diff
@@ -83,11 +83,34 @@ informational; you cannot update their values.
 `checkpointing`.`last`:::
 (object) Contains statistics about the last completed checkpoint. 
 `checkpointing`.`last`.`checkpoint`::::
-(TBD) A unique identifier for the checkpoint.
+(integer) The sequence number for the checkpoint.
 `checkpointing`.`last`.`time_upper_bound_millis`::::
-(date) TBD
+(date) When using time-based synchronization, this timestamp indicates the
+upper bound of data that is included in the checkpoint. Typically, this value
+is equal to the `checkpointing`.`last`.`time_upper_bound_millis` minus the
+`sync`.`time`.`delay`, which is defined when you create the {transform}.
```

Comment:
I would not include "Typically, this value is ..." This was just for your information, not meant to be put here.